unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java

```diff
@@ -354,7 +354,7 @@ enum Type {
         }
 
         private static final UnicodeSet PROPCHARS =
-                new UnicodeSet("[a-zA-Z0-9\\:\\-\\_\\u0020\\p{pattern white space}]");
+                new UnicodeSet("[a-zA-Z0-9.\\:\\-\\_\\u0020\\p{pattern white space}]");
```

Comment:
Note: this is so you can use something like `U5.2.0:NFKC_Casefold` in an `In` line. (I ended up not putting tests for the earliest applicable version, because we don’t have those for the other policies. We could add those later if we feel like it.)

Reply:
Makes sense